app/services/BikListService.scala

```diff
@@ -38,38 +40,39 @@ class BikListService @Inject() (
   def currentYearList(implicit
     hc: HeaderCarrier,
     request: AuthenticatedRequest[_]
-  ): Future[(Map[String, String], List[Bik])] = {
-    val response = tierConnector.genericGetCall[List[Bik]](
+  ): Future[List[Bik]] = {
+    val response = tierConnector.get(
       uriInformation.baseUrl,
       uriInformation.getRegisteredPath,
       request.empRef,
       controllersReferenceData.yearRange.cyminus1
     )
 
-    response.map { resultOption: List[Bik] =>
-      (tierConnector.pbikHeaders, resultOption.distinct)
+    response.map { resp =>
+      resp.json.as[List[Bik]].distinct
     }
   }
 
-  def nextYearList(implicit
+  def getNextYearList(implicit
```

Comment:
when we try to change the response to a List[Bik] from this method we need to call distinct where it gets used. 
forgot to do that here in this draft